pkgs/development/python-modules/pymetno/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, aiohttp
+, async-timeout
+, pytz
+, xmltodict
+}:
+
+buildPythonPackage rec {
+  pname = "PyMetno";
+  version = "0.5.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "e532544495200210407e3d68f719c271435da6f3bfe696e708b1d5d603a21948";
+  };
```

Comment:
```
running build_ext

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
Finished executing setuptoolsCheckPhase
/nix/store/lkll7y216pk69jl93kizxgmi9hnpv5wy-python3.7-PyMetno-0.5.0
```

no tests packages in pypi sdist, please fetch from github.

Tests are critical to ensuring the package isn't broken.

Reply:
Done. Unfortunately there are not tests on Github either.

Reply:
:(